mongodb_atlas/metadata.csv

```diff
@@ -95,3 +95,15 @@ mongodb.atlas.system.network.bytes.out,rate,60,byte,second,The average rate of p
 mongodb.atlas.network.bytes.in,rate,60,byte,second,The average rate of physical (after any wire compression) bytes sent to this database server per second,0,mongodb_atlas,network bytes in,
 mongodb.atlas.network.bytes.out,rate,60,byte,second,The average rate of physical (after any wire compression) bytes sent from this database server per second,0,mongodb_atlas,network bytes out,
 mongodb.atlas.network.num.requests,rate,60,request,second,The average rate of requests sent to this database server per second,0,mongodb_atlas,network num requests,
+mongodb.atlas.search.index.stats.commands.getmore,rate,60,operation,,Number of getmore commands run on all Atlas Search queries,0,mongodb_atlas,num search index getmore command,
+mongodb.atlas.search.index.stats.deletes,rate,60,operation,,Number of documents or fields that Atlas Search deleted,0,mongodb_atlas,num search index delete,
+mongodb.atlas.search.index.stats.index.fields,gauge,,operation,,Number of unique fields present in the Atlas Search index,0,mongodb_atlas,num search index fields,
+mongodb.atlas.search.index.stats.index.size,gauge,,byte,,Size of all indexes on disk,0,mongodb_atlas,size of search index,
+mongodb.atlas.search.index.stats.inserts,rate,60,operation,,Number of documents or fields that Atlas Search inserted,0,mongodb_atlas,num search index insert,
+mongodb.atlas.search.index.stats.max.replication.lag,gauge,,millisecond,,Approximate number of milliseconds that Atlas Search is behind in replicating changes from the oplog of mongod,0,mongodb_atlas,search index replication lag,
+mongodb.atlas.search.index.stats.queries.error,rate,60,operation,,Number of queries for which Atlas Search is unable to return a response,0,mongodb_atlas,num search index query error,
+mongodb.atlas.search.index.stats.queries.success,rate,60,operation,,Number of queries for which Atlas Search successfully returned a response,0,mongodb_atlas,num search index query success,
+mongodb.atlas.search.index.stats.queries.total,rate,60,operation,,Number of queries submitted to Atlas Search,0,mongodb_atlas,num search index query total,
+mongodb.atlas.search.index.stats.updates,rate,60,operation,,Number of documents or fields that Atlas Search updated,0,mongodb_atlas,num search index update,
+mongodb.atlas.search.jvm.current.memory,gauge,,operation,,Amount of memory that the JVM heap used,0,mongodb_atlas,seach jvm memory use,
```

Comment:
I can't find what unit JVM memory is in, I think it should be in bytes but looking at usage, it's very low 
<img width="1445" alt="image" src="https://github.com/DataDog/integrations-extras/assets/116591198/584f106f-e4bc-4dda-b1ec-5dbc7b107b7c">

Can't find detail in mongo doc 
https://www.mongodb.com/docs/atlas/tutorial/datadog-integration/#performance-metrics-available-to-datadog
https://www.mongodb.com/docs/atlas/review-atlas-search-metrics/